easypoll/widget/EasyPoll.jsx

```diff
@@ -55,13 +55,15 @@ const tabs = {
   },
 };
 
-const getFirstSBTToken = () => {
-  const view = Near.view("registry.i-am-human.near", "sbt_tokens_by_owner", {
-    account: `${context.accountId}`,
-    issuer: "fractal.i-am-human.near",
+//verify human with nada.bot
+function isHumanWithNadaBot() {
+  Near.asyncView("v1.nadabot.near", "is_human", {
+    account_id: context.accountId,
+  }).then((result) => {
+    console.log(result);
```

Comment:
Is this a boolean? What does "result" look like? Reply here what it console logs

Reply:
Yes it is boolean, it returns **false** for the account I tested with

Reply:
I think you are console logging a boolean result " false", but since "return result" is out of the .then scope, you are always returning "undefined", which evaluates to false.

Try with an account that should evaluate true  -- such as efiz.near

Reply:
I've made some fixes to handle the promise properly, I tested with another account that returns true

Reply:
Cool, remove the console.log and then we're good to merge

Reply:
Alright